src/orange/OrangeTrackView.hh

```diff
@@ -251,6 +269,51 @@ OrangeTrackView& OrangeTrackView::operator=(DetailedInitializer const& init)
     return *this;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * The current position.
+ */
+CELER_FUNCTION Real3 const& OrangeTrackView::pos() const
+{
+    return this->make_lsa().pos();
```

Comment:
Man I hate doing this to you but only after our discussion this morning that I realized this is wrong: we need to return the global position at level zero, not the local position at the current level. Same with direction below.

Reply:
The notion of local vs. global doesn't really exist at level 0, as it is always the root universe, which does not have a translation. So this should just be:

```
return this->make_lsa(LevelId{0}).pos();
```
correct?

Reply:
Correct.